src/core/exec-invoke.c

```diff
@@ -5010,6 +5010,11 @@ int exec_invoke(
                 return log_error_errno(SYNTHETIC_ERRNO(EINVAL), "Invalid command line arguments.");
         }
 
+        if (params->debug_wait) {
+                log_info("Blocked and waiting to attach debugger.");
```

Comment:
this should really be log_notice() i guess, it's more important that just LOG_INFO. The user must take action afterall.

also, given how relevant this message is, i'd really prefer it this was recognizable/structured message, registered in the catalog. i.e. look how log_struct() is used, and make up a MESSAGE_ID= for it, and add an entry to catalog/ to it

Reply:
also, maybe reword this a bit:

"Process will not stop and wait until a debugger is attached."